.cargo/config.toml

```diff
@@ -10,9 +10,26 @@ lto = true
 # https://blog.rust-lang.org/2023/11/16/Rust-1.74.0.html#lint-configuration-through-cargo
 rustflags = [
 	# rustc additional warnings:
-	"--warn",
-	"unused_crate_dependencies",
-	# clippy additional warnings:
-	"--warn",
-	"clippy::wildcard_imports",
+	"-Wunused_crate_dependencies",
+	# Rust 2018 idioms that are not yet warn-by-default:
+	"-Welided_lifetimes_in_paths",
+	"-Wunused_extern_crates",
+	"-Wexplicit_outlives_requirements",
+	# 📎 Lints that are enabled (warn/deny) by default
+	"-Wclippy::all",
+	# Restriction (optional, neutral lints)
+	"-Wclippy::dbg_macro",                       # Lint against leftover `dbg!` macros
```

Comment:
I think `todo` and `dbg_macro` are really useful! However, I suggest turning it on only in CI, and not during local development, especially that they show up in the editor, which makes it harder to spot actual errors/warnings.

Unfortunately, conditionally overriding `rustflags` (in CI) can only be done via command line args, so we have a central place to do it here, for example `--deny warnings`:
https://github.com/Xanewok/slang/blob/2b9527e465cf11488e63b9e5e76313307da8ff3d/crates/infra/utils/src/cargo/workspace.rs#L109-L117

Thoughts?

Reply:
Yeah, this makes sense and I agree. It'd be great to have this specified somehow in the manifest/config rather than hardcoding this in the logic but I guess it's not a big deal, since these are just 2 reasonable Clippy lints.

Reply:
Done in 301ad8ebafd2bdc8fede7336416a038178578f01